package.json

```diff
@@ -9,8 +9,9 @@
     "dev:server": "nodemon -e ts,graphql -x ts-node -r dotenv/config src",
     "playground": "graphql playground",
     "deploy": "now --public --dotenv .env.production && now alias && now rm --yes --safe moltinql",
-    "build": "rimraf dist && tsc",
-    "start": "node dist",
+    "compile": "rimraf build/dist && tsc",
+    "build": "yarn compile && cp src/types/*.graphql build/dist/types/",
```

Comment:
I'm not sure if this is the ideal approach (i.e. manually copying the graphql typedef files). Most of the examples I've seen online uses `.js` file extension for the typedefs and then exports the `type` definitions. If we could do that with typescript, I believe just running `tsc` command will be enough.

Reply:
We can improve it later down the road.